packages/esm-service-queues-app/src/types/index.ts

```diff
@@ -291,11 +291,23 @@ export interface QueueEntryPayload {
   };
 }
 
+export interface AllowedPriority {
```

Comment:
Why not use `OpenmrsResource ` ?

Reply:
Ahh... didn't know that exists. I think explicitly naming the type is still a good idea, but I'll do `type AllowedPriority = OpenmrsResource` instead.